src/common/opencl.h

```diff
@@ -599,14 +599,14 @@ static inline gboolean dt_opencl_image_fits_device(const int devid, const size_t
 {
   return FALSE;
 }
-static inline gboolean dt_opencl_buffer_fits_device(const int devid, const size_t required)
-{
-  return FALSE;
-}
 static inline size_t dt_opencl_get_device_available(const int devid)
 {
   return 0;
 }
+static inline void dt_opencl_check_device_available(const int devid)
+{
+  return;
```

Comment:
What's the purpose of a function that does nothing and returns nothing ?

Reply:
In fact i don't think we need it at all as only used in opencl code.